composer.json

```
@@ -39,11 +39,11 @@
         "ext-json": "*",
         "enlightn/security-checker": "^1.1|^2.0",
         "guzzlehttp/guzzle": "^7.0",
-        "larastan/larastan": "^2.0",
+        "larastan/larastan": "^2.0|^3.0",
         "laravel/framework": "^9.0|^10.0|^11.0",
         "nikic/php-parser": "^4.0|^5.0",
-        "phpstan/phpstan": ">=1.10.48",
-        "phpstan/phpstan-deprecation-rules": "^1.1",
+        "phpstan/phpstan": ">=1.10.48|^2.0.3",
+        "phpstan/phpstan-deprecation-rules": "^1.1|^2.0.0",
```

@HamzaAlayed you should not set the minor version to let PHPStan go to 2.1 for exemple

```suggestion
        "phpstan/phpstan": ">=1.10.48|^2.0",
        "phpstan/phpstan-deprecation-rules": "^1.1|^2.0",
```